### PR TITLE
Update SachinTest.txt

### PR DESCRIPTION
**Pull request by sachinsg108**
New code added.

### CODE REVIEW

**Comment by sachinsg108** [comment]
Please review

File: NewFile.txt [hunk 1]
```diff
@@ -1 +1,3 @@
 New file added with Report1Generate feature branch.
+
+Latest change in code for collaboration.
```
Comment (devopslearning222) [inline NewFile.txt:3]: This is not the right logic. please change it.

File: index.html [hunk 2]
```diff
@@ -5,8 +5,8 @@
 These are changes from browser.
 
 
-More changes locally. Now more changes locally again.
+Latest change in code for collaboration.
 
-This change is from server.
+Latest change in code for collaboration.
```
Comment (devopslearning222) [inline index.html:10]: Revert this code

**Review by devopslearning222 — CHANGES_REQUESTED** [review]

File: NewFile.txt (hunk 1 above)
Comment (sachinsg108) [reply NewFile.txt:3]: Done changes

File: index.html (hunk 2 above)
Comment (sachinsg108) [reply index.html:10]: Reverted changes

File: NewFile.txt (hunk 1 above)
Comment (devopslearning222) [reply NewFile.txt:3]: ok

**Review by devopslearning222 — APPROVED** [review]

**Pull request merged**